### ISSUE TITLE
Global Settings Page 404ing

### ISSUE THREAD

**Issue by natbredenkamp** [LinusEkstrom/AddOn.Episerver.Settings#35]
Added the package to a net 5 / CMS 12 site, the global settings button appears correctly on the top menu, but 404s when requested.
I had added a couple of global settings items like the below.
```
  [SettingsContentType(AvailableInEditMode = false, GUID = "12345678-ABCD-EFGH-IJKL-0987654321ABCD", SettingsInstanceGuid = "C9AE6938-CB97-4636-A4B2-6B7F0C98FFAA", SettingsName = "Block Mappings")]
  public class HeadingBlockMappings : SettingsBase
  {
public virtual IList<SelectorName> ColourMapping {get; set;}
```
Can't see what the problem is - not sure if something specific needs to be added to the startup to make this work?
noticed that the folder in the modules/_protected has no zip in it.. just a module.config - in fact its actually only added a link to the folder in there - so I get the little shortcut icon on the folder and module.config
using version 2.0.0
v 1.1.3 seems to work

**Comment by vansterhant** [comment]
Sounds like the copy task isn't working as intended, I'll have a look

**Comment by vansterhant** [comment]
@natbredenkamp, your solution of going through version 1.1.3 was clever, since that will copy the needed modules.config to the correct location :-) 
Either way the copy task was fixed for version 2.0.1
